docs/docs/deploying-to-21yunbox.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Deploying to 21YunBox
+---
+
+[21YunBox](https://www.21yunbox.com) provides blazing fast Chinese CDN, Continuous Deployment, one click HTTPS and [much more](https://www.21yunbox.com/docs/v2/index.html), providing developers a hassle-free solution to launch their web projects in China.
+
+Static sites are **insanely easy** on 21YunBox and include the following features:
```

Comment:
Again per the [style guide for documentation ](https://www.gatsbyjs.org/contributing/gatsby-style-guide/#avoid-easy-and-simple) we aviod using words like ,"easy" , "simple","insane". More inclusive language is advised 
```suggestion
Static sites are **quick to deploy** on 21YunBox and include the following features:
```

Reply:
LGTM